### DISCUSSION TITLE
osu!standard catchcircles

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#26641] [Ideas]
how these notes will work is that if your cursor is over the hitbox of the notes during the timing window that will at least give a MEH, it will count as a GREAT without requiring tapping the notes

if the note is hit, the hitsound will occur when the circle was placed or as soon as it's hit if hit late 

to make them distinct from hitcircles, they could have a hole and also sound different (but quieter as to not take attention away from the notes to be emphasized) 

the judgement mechanic of the catch notes allows new placements in certain situations such as:
representing a stream of notes without causing unnecessary difficulty spikes and sounds in the music that would otherwise not be supported by a hitcircle, slider, or spinner
allowing more creative gameplay and representation such as having the cursor dance according to the placement of catchcircles

the relax mod will not fit the use cases given since it turns the whole map into catchcircles which unless a map was designed around everything being caught, wouldn't represent what the mapper intended and more interesting patterns can be formed if a mix of the other available note types can be used with the catchcircles

**Comment by Rekunan** [comment]
There should also be catchslider and catchspinner equivalents if this gets added.
